tests/tensor_search/integ_tests/test_search_combined.py

```diff
@@ -172,6 +198,73 @@ def tearDown(self) -> None:
         super().tearDown()
         self.device_patcher.stop()
 
+    @pytest.mark.skipif(torch.cuda.is_available() is True, reason="We skip this test if we have cuda support. This model is 5gb and is very slow on g4dn.xlarge and may crash it")
+    def test_search_video(self):
+        documents = [
+            {"video_field_1": "https://marqo-k400-video-test-dataset.s3.amazonaws.com/videos/---QUuC4vJs_000084_000094.mp4", "_id": "1"},
+            # Replace the audio link with something marqo-hosted
+            {"audio_field_1": "https://audio-previews.elements.envatousercontent.com/files/187680354/preview.mp3", "_id": "2"}, 
```

Comment:
better to upload the file to our public s3 bucket just in case envato moves or deletes this file in the future.

Reply:
Yes, I'll still need to replace this audio link with one from Li's dataset